src/module_PI/Raspberry_PI/main/Camera.java

```diff
@@ -0,0 +1,64 @@
+package module_PI.Raspberry_PI.main;
+
+import java.awt.Dimension;
+import java.io.File;
+import java.io.IOException;
+
+import javax.imageio.ImageIO;
+
+import com.github.sarxos.webcam.*;
+import com.github.sarxos.webcam.ds.v4l4j.V4l4jDriver;
+
+public class Camera {
```

Comment:
I'm not sure where to add this comment so I'll just put it here. There needs to be a little documentation on this class as well as in the readme for the specifics on the camera. It should include things like how to hook the camera up, the fact that you've got the driver hardcoded so it will only run on Linux operating systems, what the camera is actually going to be used for in the functionality of the alarm, etc.

Another thought is that I don't know that we want to require a user with the Raspberry Pi module to have a Webcam. If they don't have one, how are we going to handle that? In the main PiApp just don't initialize one? Take that scenario into consideration on your next Pull Request for this camera stuff.

Neither of these need to be added to this Pull Request, unless you want them to, but should be added in the next Pull Request you make.